src/index.js

```diff
@@ -1 +1,26 @@
 'use strict';
+
+import express from 'express';
+import cors from 'cors';
+import { WebSocketServer } from 'ws';
+import { emitter } from './controllers/message.controller.js';
+import { messagesRoute } from './routes/messages.route.js';
+import { roomsRoute } from './routes/room.route.js';
+
+const app = express();
+app.use(cors());
+app.use(express.json());
+
+app.use('/messages', messagesRoute);
+app.use('/rooms', roomsRoute);
+
+const server = app.listen(3005);
+
+const wss = new WebSocketServer({ server });
+
+emitter.on('message', (message) => {
+  for (const client of wss.clients) {
+    client.send(JSON.stringify(message));
+  }
+});
```

Comment:
You are sending messages to all connected WebSocket clients whenever a new message is created. This could potentially lead to a lot of unnecessary network traffic if not all clients are interested in all messages. Consider adding some kind of filtering or topic system, so that clients can subscribe to the messages they are interested in.